tests/APIv2/AbstractAPIv2Test.php

```diff
@@ -144,7 +144,7 @@ protected function pagingTest($resourceUrl) {
         $pagingTestUrl = $resourceUrl.(strpos($resourceUrl, '?') === false ? '?' : '&').'limit=1';
         $resourcePath = parse_url($resourceUrl, PHP_URL_PATH);
 
-        $result = $this->api()->get($pagingTestUrl);
+        $result = $this->api()->get($pagingTestUrl, ['pinOrder' => 'mixed']);
```

Comment:
This is a bit of a kludge, but we know that the discussions endpoints have an optimization for pinned posts that make asserting definitive orders and paging difficult. This setting turns that off for most regular tests. The `DiscussionsTest` then has specific tests for pinned posts.